tests/topotato/test_bgp_default-originate_route-map_match.py

```diff
@@ -0,0 +1,114 @@
+from topotato import *
+
+"""
+Test if default-originate works with ONLY match operations.
+"""
+
+
+@topology_fixture()
+def allproto_topo(topo):
+    """
+    [ r1 ]
+      |
+    { s1 }
+      |
+    [ r2 ]
+
+    """
+    topo.router("r1").lo_ip4.append("172.16.255.254/32")
+    topo.router("r1").iface_to("s1").ip4.append("192.168.255.1/24")
+    topo.router("r2").iface_to("s1").ip4.append("192.168.255.2/24")
+
+
+class Configs(FRRConfigs):
+    routers = ["r1", "r2"]
+
+    zebra = """
+    #% extends "boilerplate.conf"
+    #% block main
+    #%   if router.name == 'r1'

```

Comment:
This should probably also work with an empty zebra config, the IP addrs are configured by topotato